modules/apps/object/object-test/src/testFunctional/tests/ObjectPortlet.testcase

```diff
@@ -799,7 +799,21 @@ definition {
 
 		ObjectPortlet.viewEntry(entry = "Test text");
 
-		ObjectPortlet.deleteEntry(entry = "Test text");
+		Click(
+			key_entry = "Test text",
+			locator1 = "ObjectPortlet#ENTRY_KEBAB_MENU");
+
+		ClickNoError(locator1 = "ObjectPortlet#DELETE_ENTRY_BUTTON");
+
+		AssertTextEquals(
+			locator1 = "WorkflowAllItems#DETAIL_PAGE_TITLE",
+			value1 = "Delete Entry");
+
+		AssertTextEquals(
+			locator1 = "WorkflowDefinition#DIALOG_BOX_BODY",
+			value1 = "It may affect many records. Are you sure you want to delete this entry?");
```

Comment:
None of the other tests cover the new modal, so as not to be onerous to cover it by the macro (impacting everyone), this specific test must cover it. This test has the property `testray.component.names = "Frontend Data Set"`